app/src/renderer/components/transactions/LiTransaction.vue

```diff
@@ -39,7 +39,7 @@ export default {
       default: null
     },
     time: {
-      type: Number,
+      type: [String, Number],
```

Comment:
we are not supposed to allow this

Reply:
in this case the time is not calculated, because the block is already available, so it's returned directly instead of getting the ISO time (number).

Reply:
then we should calculate the time string in the other cases so the types are more predictable

Reply:
I'd leave this as it is because removing it breaks the transaction dates

Reply:
no, let's please change it in this PR. we agreed that we don't have several types per prop.

Reply:
updated to be `ISOString`